src/WebJobs.Script/Diagnostics/FileTraceWriter.cs

```diff
@@ -32,10 +33,11 @@ public class FileTraceWriter : TraceWriter, IDisposable
         private Timer _flushTimer;
         private ConcurrentQueue<string> _logBuffer = new ConcurrentQueue<string>();
 
-        public FileTraceWriter(string logFilePath, TraceLevel level) : base(level)
+        public FileTraceWriter(string logFilePath, TraceLevel level, Func<string, string> messageFormatter = null) : base(level)
```

Comment:
FileTraceWriter is a public type and we just added an optional param to the constructor.. do we care about this runtime breaking change? I know script packages are still not RTM so from a semantic versioning perspective we're OK, but just wanted to double check anyway...